distribution/build.gradle

```diff
@@ -220,6 +220,12 @@ project.rootProject.subprojects.findAll { it.parent.path == ':modules' }.each {
 // use licenses from each of the bundled xpack plugins
 Project xpack = project(':x-pack:plugin')
 xpack.subprojects.findAll { it.parent == xpack }.each { Project xpackModule ->
+// Ugly hacks to see what is missing from integ test. We need core and security because we need to call to ConfigInitialNode on Startup to
```

Comment:
I'll raise a separate PR to wrap this as a proper change for the INTEG_TEST distribution

Reply:
opened https://github.com/elastic/elasticsearch/pull/77632